tests/e2e/stateapp/stateapp_test.go

```diff
@@ -247,3 +284,96 @@ func TestStateApp(t *testing.T) {
 		})
 	}
 }
+
+func generateStateTransactionCases() []testStateTransactionCase {
+	testCase1Key, testCase2Key := guuid.New().String(), guuid.New().String()
+	testCase1Value := "The best song ever is 'Highwayman' by 'The Highwaymen'."
+	testCase2Value := "Hello World"
+	// Just for readability
+	emptyResponse := requestResponse{
+		nil,
+	}
+
+	testStateTransactionCases := []testStateTransactionCase{
+		{
+			"Test state transaction APIs for single transactions",
+			[]stateTransactionTestStep{
+				{
+					"transact",
+					newStateTransactionRequestResponse(utils.StateTransactionKeyValue{testCase1Key, testCase1Value, "upsert"}),
+					emptyResponse,
+				},
+				{
+					"get",
```

Comment:
Why do we need to test "get" ? isn't this test for single operation "get"?

Reply:
I wanted to use it to check whether the key was successfully upserted

Reply:
Thanks for the explanation. ok this is good :) is there a reason why you removed this command in the latest change?

Reply:
I added it back